globals.go

```diff
@@ -276,20 +279,22 @@ var InternalAliases []string = []string{
 	GitTagAlias,
 }
 
-var TaskFailureStatuses []string = []string{
+var TaskSystemFailureStatuses []string = []string{
```

Comment:
This doesn't need the `[]string` type annotation because it's inferred to be a string slice from its initialized value. Can you remove it (and for the other `var VARNAME []string` usages in the file)?

Reply:
I would avoid calling this `TaskSystemFailureStatuses` because there's already `TaskSystemFailed`, which generally means a purple task, whereas `TaskSetupFailed` is lavendar and understood to be distinct from a system failure.

Maybe something like `TaskNonGenericFailureStatuses`?

Reply:
Can you add a doc comment?